### ISSUE TITLE
[rank0]:[W706 07:35:33.924205988 ProcessGroupNCCL.cpp:1476]

### ISSUE THREAD

**Issue by keruoya** [GaParmar/img2img-turbo#149]
ipex flag is deprecated, will be removed in Accelerate v1.10. From 2.7.0, PyTorch has all needed optimizations for Intel CPU and XPU.
Using RTX 4000 series which doesn't support faster communication speedups. Ensuring P2P and IB communications are disabled.
/home/wl/anaconda3/envs/torch110cp39/lib/python3.9/site-packages/clip/clip.py:6: UserWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html. The pkg_resources package is slated for removal as early as 2025-11-30. Refrain from using this package or pin to Setuptools<81.
  from pkg_resources import packaging
/home/wl/anaconda3/envs/torch110cp39/lib/python3.9/site-packages/clip/clip.py:6: UserWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html. The pkg_resources package is slated for removal as early as 2025-11-30. Refrain from using this package or pin to Setuptools<81.
  from pkg_resources import packaging
{'variance_type', 'thresholding', 'dynamic_thresholding_ratio', 'clip_sample_range', 'rescale_betas_zero_snr'} was not found in config. Values will be initialized to default values.
{'mid_block_add_attention', 'latents_mean', 'shift_factor', 'use_quant_conv', 'use_post_quant_conv', 'latents_std'} was not found in config. Values will be initialized to default values.
All model checkpoint weights were used when initializing AutoencoderKL.

All the weights of AutoencoderKL were initialized from the model checkpoint at stabilityai/sd-turbo.
If your task is similar to the task the model of the checkpoint was trained on, you can already use AutoencoderKL for predictions without further training.
All model checkpoint weights were used when initializing UNet2DConditionModel.

All the weights of UNet2DConditionModel were initialized from the model checkpoint at stabilityai/sd-turbo.
If your task is similar to the task the model of the checkpoint was trained on, you can already use UNet2DConditionModel for predictions without further training.
Initializing model with random weights
Initializing model with random weights
Setting up [LPIPS] perceptual loss: trunk [vgg], v[0.1], spatial [off]
/home/wl/anaconda3/envs/torch110cp39/lib/python3.9/site-packages/torchvision/models/_utils.py:208: UserWarning: The parameter 'pretrained' is deprecated since 0.13 and may be removed in the future, please use 'weights' instead.
  warnings.warn(
/home/wl/anaconda3/envs/torch110cp39/lib/python3.9/site-packages/torchvision/models/_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=VGG16_Weights.IMAGENET1K_V1`. You can also use `weights=VGG16_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
Setting up [LPIPS] perceptual loss: trunk [vgg], v[0.1], spatial [off]
/home/wl/anaconda3/envs/torch110cp39/lib/python3.9/site-packages/torchvision/models/_utils.py:208: UserWarning: The parameter 'pretrained' is deprecated since 0.13 and may be removed in the future, please use 'weights' instead.
  warnings.warn(
/home/wl/anaconda3/envs/torch110cp39/lib/python3.9/site-packages/torchvision/models/_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=VGG16_Weights.IMAGENET1K_V1`. You can also use `weights=VGG16_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
Loading model from: /home/wl/anaconda3/envs/torch110cp39/lib/python3.9/site-packages/lpips/weights/v0.1/vgg.pth
Loading model from: /home/wl/anaconda3/envs/torch110cp39/lib/python3.9/site-packages/lpips/weights/v0.1/vgg.pth
/home/wl/anaconda3/envs/torch110cp39/lib/python3.9/site-packages/torch/_compile.py:51: UserWarning: optimizer contains a parameter group with duplicate parameters; in future, this will cause an error; see github.com/pytorch/pytorch/issues/40967 for more information
  return disable_fn(*args, **kwargs)
/home/wl/anaconda3/envs/torch110cp39/lib/python3.9/site-packages/torch/_compile.py:51: UserWarning: optimizer contains a parameter group with duplicate parameters; in future, this will cause an error; see github.com/pytorch/pytorch/issues/40967 for more information
  return disable_fn(*args, **kwargs)
Steps:   0%|                                                                                                                                                                | 0/10000 [00:00<?, ?it/s]/home/wl/anaconda3/envs/torch110cp39/lib/python3.9/site-packages/torch/autograd/graph.py:824: UserWarning: Grad strides do not match bucket view strides. This may indicate grad was not created according to the gradient layout contract, or that the param's strides changed since DDP was constructed.  This is not an error, but may impair performance.
grad.sizes() = [512, 4, 1, 1], strides() = [4, 1, 4, 4]
bucket_view.sizes() = [512, 4, 1, 1], strides() = [4, 1, 1, 1] (Triggered internally at /pytorch/torch/csrc/distributed/c10d/reducer.cpp:328.)
  return Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass
/home/wl/anaconda3/envs/torch110cp39/lib/python3.9/site-packages/torch/autograd/graph.py:824: UserWarning: Grad strides do not match bucket view strides. This may indicate grad was not created according to the gradient layout contract, or that the param's strides changed since DDP was constructed.  This is not an error, but may impair performance.
grad.sizes() = [512, 4, 1, 1], strides() = [4, 1, 4, 4]
bucket_view.sizes() = [512, 4, 1, 1], strides() = [4, 1, 1, 1] (Triggered internally at /pytorch/torch/csrc/distributed/c10d/reducer.cpp:328.)
  return Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass
/home/wl/anaconda3/envs/torch110cp39/lib/python3.9/site-packages/torch/functional.py:554: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at /pytorch/aten/src/ATen/native/TensorShape.cpp:4314.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
/home/wl/anaconda3/envs/torch110cp39/lib/python3.9/site-packages/torch/functional.py:554: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at /pytorch/aten/src/ATen/native/TensorShape.cpp:4314.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
Steps:   0%|                                                                           | 1/10000 [00:03<8:31:38,  3.07s/it, lossD=2.12, lossG=1.09, loss_clipsim=0.783, loss_l2=1.42, loss_lpips=4.02]wandb: WARNING Data passed to `wandb.Image` should consist of values in the range [0, 255], image data will be normalized to this range, but behavior will be removed in a future version of wandb.
MLflowTracker is attempting to log a value of "[<wandb.sdk.data_types.image.Image object at 0x7f65b00c5910>, <wandb.sdk.data_types.image.Image object at 0x7f66069324f0>]" of type <class 'list'> for key "train/source" as a metric. MLflow's log_metric() only accepts float and int types so we dropped this attribute.
MLflowTracker is attempting to log a value of "[<wandb.sdk.data_types.image.Image object at 0x7f6606932760>, <wandb.sdk.data_types.image.Image object at 0x7f65b00b5af0>]" of type <class 'list'> for key "train/target" as a metric. MLflow's log_metric() only accepts float and int types so we dropped this attribute.
MLflowTracker is attempting to log a value of "[<wandb.sdk.data_types.image.Image object at 0x7f65b00b5eb0>, <wandb.sdk.data_types.image.Image object at 0x7f65b014e310>]" of type <class 'list'> for key "train/model_output" as a metric. MLflow's log_metric() only accepts float and int types so we dropped this attribute.
Steps:   0%|                                                                          | 10/10000 [00:11<3:13:38,  1.16s/it, lossD=2.03, lossG=1.09, loss_clipsim=0.776, loss_l2=1.42, loss_lpips=3.88]
[rank0]:[W706 07:35:33.924205988 ProcessGroupNCCL.cpp:1476] Warning: WARNING: destroy_process_group() was not called before program exit, which can leak resources. For more info, please see https://pytorch.org/docs/stable/distributed.html#shutdown (function operator())
